src/stories/TransactionItemComponent.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import type {Meta, StoryFn} from '@storybook/react';
+import React from 'react';
+import TransactionItemComponent from '@components/TransactionItemComponent';
+import type Transaction from '@src/types/onyx/Transaction';
+
+type TransactionItemComponentStory = StoryFn<typeof TransactionItemComponent>;
+
+const transactionItemList : Array<Transaction&{mcc:string,modifiedMCC:string}> = [
```

Comment:
are the names `mcc` already used in app and we have to follow this naming?

Reply:
mcc propoerty comes straight from onyx